src/features/rewards/tipsMigrationAlert/style.ts

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this file,
+* You can obtain one at http://mozilla.org/MPL/2.0/.
+*/
+
+import styled from 'styled-components'
+
+export const StyledWrapper = styled<{}, 'div'>('div')`
+  width: 100%;
+  display: flex;
+  background: #E9F0FF;
+  padding: 5px 30px;
+  border-bottom-left-radius: 6px;
+  border-bottom-right-radius: 6px;
+`
+
+export const StyledAlertIcon = styled<{}, 'div'>('div')`
+  width: 57px;
+  height: 57px;
+  color: #15A4FA;
+  display: inline-block;
```

Comment:
you don't need inline-block anymore now that you have flex, please remove it in all elements

Reply:
@NejcZdovc done